File: src/rentcar/Kendaraan.java
package rentcar;

public class Kendaraan {
    private String merk;
    private String tipe;
    private double tarifPerHari;
    private String statusKendaraan;

    public Kendaraan(String merk, String tipe, double tarifPerHari) {
        this.merk = merk;
        this.tipe = tipe;
        this.tarifPerHari = tarifPerHari;
        this.statusKendaraan = "Tersedia";
    }

    public String getMerk() {
        return merk;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public String getTipe() {
        return tipe;
    }

    public void setTipe(String tipe) {
        this.tipe = tipe;
    }

    public double getTarifPerHari() {
        return tarifPerHari;
    }

    public void setTarifPerHari(double tarifPerHari) {
        this.tarifPerHari = tarifPerHari;
    }

    public String getStatusKendaraan() {
        return statusKendaraan;
    }

    public void setStatusKendaraan(String statusKendaraan) {
        this.statusKendaraan = statusKendaraan;
    }
    
   
}

File: RentCarCpp/kendaraan.cpp
#include "kendaraan.h"

Kendaraan::Kendaraan(QString m, QString t, double tarif) {
    merk = m;
    tipe = t;
    tarifPerHari = tarif;
    isTersedia = true;
}

Kendaraan::Kendaraan() {
    merk = "";
    tipe = "";
    tarifPerHari = 0;
    isTersedia = true;
}
